raiden/accounts.py

```diff
@@ -19,6 +20,10 @@ class InvalidAccountFile(Exception):
     pass
 
 
+class KeystoreFileNotFound(RaidenError):
```

Comment:
If this inherited from `ClickException` or one of its subclasses, you could add the error message and its formatting into the `show` method of the class (http://click.palletsprojects.com/en/5.x/exceptions/#which-exceptions-exist). Not sure it that really is an improvement in this case, I just want to point out that it exists.